etl/steps/data/garden/demography/2022-12-08/population/meta.yml

```diff
@@ -5,10 +5,14 @@ dataset:
   description: |
     This dataset has been created using multiple sources:
 
-      - We use UN WPP 2022 for data prior to 1950.
-      - Before that, we combine HYDE and Gapminder.
+    - UN WPP (2022): Covers the period 1950-2100. Future projections are based on the Medium variant.
+    - Gapminder (v6): Mostly covers the period 1800-1949. In addition, it complements the dataset with population data for the "Vatican" until 2100 (it is missing in UN WPP's estimates).
+    - HYDE (v3.2): Mostly covers the period 10,000 BCE - 1799. In addition, it complements the dataset with population for
+    some countries outside of this year period. E.g. it provides data for "Svalbard and Jan Mayen" and "Netherlands Antilles" until 2017.
```

Comment:
Netherland Antilles dissolved in 2010, so this paragraph may cause some confusion.